test/fixtures/db_definitions/mysql.sql

```diff
@@ -52,7 +52,7 @@ create table product_tariffs (
 );
 
 create table suburbs (
-    city_id int not null,
+    city_id int not null auto_increment,
```

Comment:
Suspect we don't need this change as we are skipping the test.

Reply:
This is used in other tests which will fail without it.